app/connectors/model/UploadParameters.scala

```diff
@@ -18,5 +9,6 @@ case class UploadParameters(
   additionalMetadata: Map[String, String],
   contentLengthRange: ContentLengthRange,
   expectedContentType: Option[String],
-  successRedirect: Option[String]
+  successRedirect: Option[String],
```

Comment:
Documentation says that these file are mandatory - here are declared as optional

Reply:
Sorry, my misunderstanding.